lib/PuppeteerSharp/Helpers/EnumHelper.cs

```diff
@@ -77,4 +78,39 @@ public static string ToValueString<TEnum>(this TEnum value)
 
         return enumValues[value];
     }
+
+    public static TEnum FromValueString<TEnum>(string value)
+        where TEnum : struct, Enum
+    {
+        var enumValues = _stringToEnumCache.GetOrAdd(typeof(TEnum), type =>
+        {
+            var names = Enum.GetNames(type);
+            var dictionary = new Dictionary<string, Enum>(StringComparer.OrdinalIgnoreCase);
+            foreach (var valueName in names)
+            {
+                var field = type.GetField(valueName);
+                var fieldValue = (TEnum)field.GetValue(null);
+                dictionary[valueName] = fieldValue;
+                if (field.GetCustomAttribute<EnumMemberAttribute>()?.Value is { } enumMember)
+                {
+                    dictionary[enumMember] = fieldValue;
+                }
+            }
+
+            return dictionary;
+        });
+
+        var defaultEnumAttribute = typeof(TEnum).GetCustomAttribute<DefaultEnumValueAttribute<TEnum>>();
```

Comment:
Can this call to `GetCustomAttribute` be moved to after the lookup in `enumValues`?